### DISCUSSION TITLE
6.6.0 (20583) does not seem to register dial presses

### DISCUSSION THREAD

**Discussion by markcellis** [elgatosf/elgato#114] [Help] [answered]
My customers are reporting that Stream Deck Plus dial presses are no longer working with the 6.6 version. Is anyone else seeing this? I have not tried to look into what is going on with the debugger yet.

**Comment by ForgetfulMemoryFoam** [comment]
With the newest Stream Deck update, I haven't seen an issue with our SD+ and the dial commands. I was testing a new plugin thoroughly with a short press, and long press that I implemented without issue. 

Are the presses for you not working at all?

**Reply by markcellis** [reply, marked as answer]
The problem was on my end. Elgato had removed the DialPress event since they now only support dial_down and dial_up. I just needed to add those two new events.
